### PR TITLE
Added info on how to move "quick connect" connections

### PR DESCRIPTION
**Pull request by mcinp**
## Changes:
I stumbled upon the problem of moving quick connect connections from Sequel Pro to Sequel Ace. This notion in the readme addresses this issue. This PR includes changes mentioned by @Kaspik from previous PR.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -29,6 +29,16 @@ To install an unoffical community maintained [Homebrew](https://brew.sh) [Cask](
 brew cask install sequel-ace
 ```
 
+### Moving saved connection list from Sequel Pro
+
+To move your quick connect list from Sequel Pro to Sequel Ace just copy the file
```
Comment (Kaspik) [inline readme.md:34]: Should we just add info and pointer to separate page that we have? https://sequel-ace.com/get-started/migrating-from-sequel-pro.html

Comment (mcinp) [reply readme.md:34]: Well yes, this link would be useful, especially that it's not directly accessible from https://sequel-ace.com/, only through  FAQ, that is not immediately visible.

Comment (Kaspik) [reply readme.md:34]: Let's update this with something like `For more migration info, check [migration guide](https://sequel-ace.com/get-started/migrating-from-sequel-pro.html)` then? :)

Comment (mcinp) [reply readme.md:34]: So maybe just 

### Moving saved connection list from Sequel Pro
And the link here

Comment (Kaspik) [reply readme.md:34]: Yeah something like that

**Pull request closed without merging**